.taskcluster.yml

```diff
@@ -43,24 +43,24 @@ tasks:
           export DOCS_PROJECT=docker-worker DOCS_TIER=workers DOCS_FOLDER=docs DOCS_README=README.md &&
           upload-project-docs
     metadata:
-      name: "Publish docker-worker docs to https://docs.taskcluster.net/reference/workers/docker-worker"
-      description: "Upload docker-worker documentation to taskcluster [docs site](https://docs.taskcluster.net/reference/workers/docker-worker)."
+      name: "Publish docker-worker docs to https://docs.community-tc.services.mozilla.com/reference/workers/docker-worker"
+      description: "Upload docker-worker documentation to taskcluster [docs site](https://docs.community-tc.services.mozilla.com/reference/workers/docker-worker)."
       owner: '{{ event.head.user.email }}'
       source: '{{ event.head.repo.url }}'
```

Comment:
This whole task actually doesn't do anything anymore, and could be removed (but again doesn't hurt)